### PR TITLE
feat: Add ruby request/response code samples

### PR DESCRIPTION
**Pull request by andrii-balitskyi**
- **Add createRubyRequest and createRubyResponse**
- **Update test snapshots**

### CODE REVIEW

File: src/lib/code-sample/ruby.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  const responseRubyClassName = pascalCase(responseKey)
+  const responseRubyParams = Object.entries(responseValue)
+    .map(([paramKey, paramValue]) => {
+      const formattedValue =
+        paramValue === null ? 'nil' : JSON.stringify(paramValue)
+      return `  ${snakeCase(paramKey)}=${formattedValue}`
+    })
+    .join('\n')
+
+  return `<Seam::${responseRubyClassName}:0x00438\n${responseRubyParams}>`
```
Comment (razor-x) [inline src/lib/code-sample/ruby.ts:41]: Where does `0x00438` come from?

**Review by razor-x — APPROVED** [review]

File: src/lib/code-sample/ruby.ts (hunk 1 above)
Comment (andrii-balitskyi) [reply src/lib/code-sample/ruby.ts:41]: The `0x00438` is the unique memory address of the object, which is an instance of the Seam class. 
Do we need to randomize it each time? For a sample, I thought the static address is sufficient, but I'm open to generating it.

Comment (razor-x) [reply src/lib/code-sample/ruby.ts:41]: Sure. Maybe just set it to something less random, like 0x00001 or 0x00000 just so it appears insignificant

Comment (andrii-balitskyi) [reply src/lib/code-sample/ruby.ts]: 95929253b3d5ed66ce272e50325c72a7a8bc11a5

**Pull request merged**